Headers/GNUstepBase/GSConfig.h.in

```diff
@@ -299,7 +299,7 @@ typedef	struct {
 #endif
 
 #if defined(_WIN32)
-#  define BOOL WinBOOL
```

Comment:
Pretty sure `WinBOOL` is correct, at least under MinGW64 and MSVC.

The MSVC CI build is failing with `WINBOOL`:
> error: unknown type name 'WinBOOL'; did you mean 'WINBOOL'?

Reply:
Interesting. The error I get on mingw64 is the exact opposite: 

`error: unknown type name 'WINBOOL'; did you mean 'WinBOOL'?
 `

Reply:
Works fine under MinGW64 with and without your change on CI though – wonder what’s different in your setup?
https://github.com/gnustep/libs-base/runs/5688864693?check_suite_focus=true